Cargo.toml

```diff
@@ -16,8 +16,8 @@ repository = { workspace = true }
 license = { workspace = true }
 
 [workspace.dependencies]
-histogram = "0.7.4"
-serde = "1.0.185"
+histogram = { version = "0.8.0" }
```

Comment:
Just `histogram = "0.8.0"` should be sufficient here?

Reply:
Fixed.